backend/README.md

```diff
@@ -0,0 +1,45 @@
+# Pathways Backend
```

Comment:
I know you were probably planning on expanding this but can we add instructions on how to install/run. Perhaps also link to the wiki page I made: https://github.com/mcode/pathway-builder/wiki/Backend-API

Reply:
Ya I'll add those, just haven't gotten around to it yet.  Good call on the wiki.